release-schema.json

```diff
@@ -29,6 +29,14 @@
             "string",
             "null"
           ]
+        },
+        "features": {
+          "title": "Features",
+          "description": "The national rules that apply to the procedure.",
```

Comment:
Let's use the longer description from eForms:

> The main features of the procedure (e.g. description of the individual stages) and information about where the full rules for the procedure can be found.

Reply:
Done